systemd/pyvast-threatbus.service

```
@@ -0,0 +1,39 @@
+[Unit]
+Description=PyVAST-Threat Bus - Connect VAST with Threat Bus
+Wants=network-online.target
+After=network-online.target
+
+[Service]
+Type=simple
+
+# user + privileges
+# We explicitly don't use DynamicUser because that can lead to recursive `chown`s.
+# Doing that is pretty slow on some file systems (e.g., xfs).
+User=threatbus
+Group=threatbus
+NoNewPrivileges=yes
```

Threatbus doesn't have a lot of persisted state, so `DynamicUser` might be viable. It has the advantage that the preparation instructions could be shortened because the `threadbus` user would not be necessary.
```suggestion
DynamicUser=yes
LogsDirectory=pyvast-threatbus
```